File: app/src/main/java/com/example/brochures/network/parsingobjects/Content__1.java
package com.example.brochures.network.parsingobjects;


import java.util.List;

public class Content__1 {

    @com.squareup.moshi.Json(name = "title")
    private String title;
    @com.squareup.moshi.Json(name = "publisherId")
    private String publisherId;
    @com.squareup.moshi.Json(name = "publisherImage")
    private String publisherImage;
    @com.squareup.moshi.Json(name = "items")
    private List<Item> items = null;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getPublisherId() {
        return publisherId;
    }

    public void setPublisherId(String publisherId) {
        this.publisherId = publisherId;
    }

    public String getPublisherImage() {
        return publisherImage;
    }

    public void setPublisherImage(String publisherImage) {
        this.publisherImage = publisherImage;
    }

    public List<Item> getItems() {
        return items;
    }

    public void setItems(List<Item> items) {
        this.items = items;
    }

}

File: app/src/main/java/com/example/brochures/network/parsingobjects/Content__1.kt
package com.example.brochures.network.parsingobjects

import com.squareup.moshi.Json

/**
 * TODO
 * @author Mikhail Avdeev (mvavdeev@sberbank.ru)
 */

class Content__1 {
    @Json(name = "title")
    var title: String? = null

    @Json(name = "publisherId")
    var publisherId: String? = null

    @Json(name = "publisherImage")
    var publisherImage: String? = null

    @Json(name = "items")
    var items: List<Item>? = null
}